.github/workflows/greetings.yml

```diff
@@ -0,0 +1,14 @@
+name: Greetings
+
+on: [pull_request]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        pr-message: 'Hello there first time contributor! Welcome to the MDAnalysis community! We ask that all contributors abide by our Code of Conduct (https://www.mdanalysis.org/pages/conduct/) and that first time contibutors introduce themselves on the developer mailing list (https://groups.google.com/g/mdnalysis-devel) so we can get to know you.'
```

Comment:
```suggestion
        pr-message: 'Hello there first time contributor! Welcome to the MDAnalysis community! We ask that all contributors abide by our [Code of Conduct](https://www.mdanalysis.org/pages/conduct/) and that first time contributors introduce themselves on the [developer mailing list](https://groups.google.com/g/mdnalysis-devel) and the [MDAnalysis Discord Server](https://discord.com/channels/807348386012987462/) so we can get to know you. Please also add yourself to `package/AUTHORS` as part of this PR.'
```

Reply:
For the Discord server I wouldn't link directly because that link is useless if you haven't been invited. https://www.mdanalysis.org/#participating has the discord information.